### ISSUE TITLE
Order of tests incorrect in CLI output

### ISSUE THREAD

**Issue by gerukin** [AnWeber/httpyac#237]
Hi Andreas, thank you for your hard work on this repo, it's really become full featured and well documented... and you're doing it all on your own as far as I can tell! 🙇‍♂️

That being said, here's a quick bug report so you don't get bored...

## Directory structure
```
|_ env
|_ requests
| |_ bug
|   |_ test.http
|_ tests
  |_ index.js
  |_ status.js
```

### requests/bug/test.http
```
{{
  module.exports = require('../../tests/')
}}

### Test 1
https://httpbin.org/status/200

{{
  tests.status.is(test, response, { val: 200 })
}}

### Test 2
https://httpbin.org/status/400

{{
  tests.status.is(test, response, { val: 400 })
}}

### Test 3
https://httpbin.org/status/500

{{
  tests.status.is(test, response, { val: 403 })
}}
```

### tests/index.js
```javascript
module.exports.tests = {
  status: require('./status.js'),
  schema: require('./schema.js')
}
```

### tests/status.js
```javascript
const run = (test, condition, errMsg) => {
  test('Status is valid', () => {
    if (condition) return true

    const error = new Error(errMsg)
    error.name = 'STATUS_ERR'
    throw error
  })
}

module.exports.is = (test, response, { val }) => {
  run(test, response.statusCode === val, `Response status is ${response.statusCode} instead of ${val}`)
}

// other status test types...
```

## Output

Running:
```
httpyac --env local --all ./requests/bug --output short
```

<img width="676" alt="Screen Shot 2022-03-31 at 17 09 40" src="https://user-images.githubusercontent.com/36127103/161012998-3edbcb7e-4e48-4abd-a0bb-200a7e2a4147.png">

🤷‍♂️

Note: it all works fine in vs-code.

**Comment by AnWeber** [comment]
Nice to hear from you again. I'm glad you still like my little hobby project. Alone is not quite right. I always find people who provide good input or some PR. Your input regarding Cli Tests was very helpful to me.
The tests are executed before the response is issued. So that the order is correct, I now simply cache the complete console output and output it after the response. I have checked it with the simplified test

```

GET https://httpbin.org/status/200

{{
  test.status(200);
}}



GET https://httpbin.org/status/400

{{
  test.status(400);
}}


GET https://httpbin.org/status/500

{{
  test.status(403);
}}

```

**Comment by AnWeber** [comment]
fix released with v5.3.0

**Comment by gerukin** [comment]
As always, thank you! Looks like it's fixed on my end too.
